### ISSUE TITLE
Add validation on select fieldtype options in blueprint builder

### ISSUE THREAD

**Issue by jesseleite** [statamic/cms#346]
So that submitting this...

![Screen Shot on 2019-04-03 at 15-43-48](https://user-images.githubusercontent.com/5187394/55509549-38055700-562a-11e9-8c80-07c1379d46e5.png)

Doesn't generate derpy stuff like this...

```yaml
-
  handle: select
  field:
    options:
      option-value: 'Option Text'
      another-option-value: 'Another Option Text'
      Hallo: null
      '': Wot
```

**Comment by jesseleite** [comment]
Still legit, but low priority.

**Comment by jasonvarga** [comment]
This should be easier to add once #3989 is merged. It adds validation, we probably just need to add the rules to the field.
